### PR TITLE
Update EIP-1: Add adoptable field

### PR DESCRIPTION
**Pull request by SamWilsn**
Reopening the discussion from #5463.

I merged without waiting for sufficient consensus, so I've reverted that PR, and opened this one.

### CODE REVIEW

**Review by Pandapip1 — DISMISSED** [review]

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by lightclient** [comment]
I don't think this provides much value. It should simply be left up to editor discretion if a stagnant EIP can be reassigned. And I generally don't think that should happen unless the original author agrees.

**Comment by Pandapip1** [comment]
> I generally don't think that should happen unless the original author agrees.

This flag is basically the author in advance stating "Yes, I am okay with someone else taking over this proposal."

**Comment by Pandapip1** [comment]
Also - this flag is optional. Anyone can choose to not have their EIP adoptable.

**Comment by SamWilsn** [comment]
I am, as usual, torn on this proposal.

I dislike reassigning proposals to new authors, but I equally dislike abandoning a well-known EIP number just because the authors can't be reached.

I would feel more comfortable if the authors opt-in to the adoption process over just reassigning without their permission.

**Comment by Pandapip1** [comment]
> I would feel more comfortable if the authors opt-in to the adoption process over just reassigning without their permission.

Should the adoptable field be by default set to false then?

**Comment by Pandapip1** [comment]
@SamWilsn @lightclient is this better?

**Comment by SamWilsn** [comment]
> > I would feel more comfortable if the authors opt-in to the adoption process over just reassigning without their permission.
> 
> Should the adoptable field be by default set to false then?

IMO, if the `adoptable` field is missing, the EIP should not be reassigned without permission but the EIP template should have `adoptable: true`.

**Comment by Pandapip1** [comment]
> IMO, if the adoptable field is missing, the EIP should not be reassigned without permission but the EIP template should have adoptable: true.

I agree with this. This was in fact my original idea. I'm not sure how it ended up being an "if it's missing then it defaults to true" situation, but it's fixed now.

File: eip-template.md
Comment (xinbenlv) [inline eip-template.md:12]: I am in favor of having an adoptable field, but I suggest keeping the template default as `false` until we see a trend that authors intentionally choose to flip it to `true`.  I worry if we create a field and default to true, the new comers of EIP authors will think it's a community etiquette to sign it off. Just like the "[ ] I agree " box in many terms of services.

Comment (xinbenlv) [inline eip-template.md:12]: I am in favor of having an adoptable field, but I suggest keeping the template default as `false` until we see a trend that authors intentionally choose to flip it to `true`.  I worry if we create a field and default to true, the new comers of EIP authors will think it's a community etiquette to sign it off. Just like the "[ ] I agree " box in many terms of services. 
```suggestion
adoptable: false  # It is recommended that you keep this as true. See EIP-1#adoptable-header for more information.
```

Comment (Pandapip1) [reply eip-template.md:12]: Uh, it should be etiquette. Nobody likes stagnant EIPs.

**Comment by Pandapip1** [comment]
> I prefer to let EIPs be "rehomed" solely at the discretion of the editors.

This enables this :)

Previously, it requires both editor and author approval. If EIPs opt-in to adoption, then only editors are required. But editors are obviously still required.

**Comment by lightclient** [comment]
It doesn't require authors approval. The editors can already unilaterally reassign an EIP. We haven't done this before and we try to get author consent, but we do have the ability to reassign.

**Comment by xinbenlv** [comment]
@lightclient I know that is for the good intention of community. 

but I respectively disagree with the assertion that editors as a group (individually or together) shall have the power to unilaterally reassign EIP. I think editors reserve the power to decline an EIP to be published, but reassign right seems beyond its governing power.

**Comment by Pandapip1** [comment]
I agree with @xinbenlv here. Adoptability should either be opt-in or opt-out, but some control should be given to authors.

**Comment by Pandapip1** [comment]
Also, I'm not sure what's up with the stagnant bot. Why wasn't it reminding us?

**Comment by Pandapip1** [comment]
Would still like this.

**Comment by lightclient** [comment]
As I've said multiple times in this thread and on EIPIP, I don't think this is a good idea. I would rather leave it up to editor discretion.

**Comment by Pandapip1** [comment]
Dismissing stale bot.

**Comment by lightclient** [comment]
I don't understand why you're dismissing the stale bot. I don't think this should be merged. Are we just going to have it sit here open forever?

**Comment by Pandapip1** [comment]
> I don't understand why you're dismissing the stale bot. I don't think this should be merged. Are we just going to have it sit here open forever?

I want to remind everybody that this still exists, and that I am still actively interested in pursuing it.

**Comment by Pandapip1** [comment]
Bump!

**Comment by Pandapip1** [comment]
Bumpity boop!

**Comment by Pandapip1** [comment]
Since enough people objected, I've set the adoptable entry in the template to be default-false, so that new EIPs are opt-in instead of opt-out.

This is a reminder: existing EIPs are, and will always be opt-in.

**Review by mspaansen — APPROVED** [review]

**Comment by Pandapip1** [comment]
Still would like this.

**Comment by Pandapip1** [comment]
Would like to reopen this.

**Comment by Pandapip1** [comment]
Bump...

**Comment by SamWilsn** [comment]
I thought we had decided we weren't doing this?

**Comment by lightclient** [comment]
I also thought we agreed not to do this

**Comment by Pandapip1** [comment]
> I thought we had decided we weren't doing this?

I thought I had simply stopped bringing it up. I probably misremembered then. Closing.

**Comment by Pandapip1** [comment]
FWIW I would still like this

**Pull request closed without merging**
